v2/go.mod

```diff
@@ -3,23 +3,23 @@ module github.com/rebuy-de/rebuy-go-sdk/v2
 go 1.15
 
 require (
-	github.com/aws/aws-sdk-go v1.34.3
+	github.com/aws/aws-sdk-go v1.35.1
 	github.com/gemnasium/logrus-graylog-hook/v3 v3.0.2
 	github.com/google/go-github/v32 v32.1.0
-	github.com/google/uuid v1.1.1
-	github.com/goreleaser/nfpm v1.6.0
+	github.com/google/uuid v1.1.2
+	github.com/goreleaser/nfpm v1.8.0
 	github.com/gorilla/securecookie v1.1.1
-	github.com/gorilla/sessions v1.2.0
+	github.com/gorilla/sessions v1.2.1
 	github.com/julienschmidt/httprouter v1.3.0
 	github.com/mitchellh/mapstructure v1.3.3
 	github.com/pkg/errors v0.9.1
 	github.com/pmezard/go-difflib v1.0.0
-	github.com/sirupsen/logrus v1.6.0
+	github.com/sirupsen/logrus v1.7.0
 	github.com/spf13/cobra v1.0.0
-	github.com/tidwall/pretty v1.0.1
-	golang.org/x/oauth2 v0.0.0-20200107190931-bf48bf16ab8d
-	golang.org/x/sync v0.0.0-20200625203802-6e8e738ad208
-	golang.org/x/tools v0.0.0-20200812231640-9176cd30088c
+	github.com/tidwall/pretty v1.0.2
+	golang.org/x/oauth2 v0.0.0-20200902213428-5d25da1a8d43
+	golang.org/x/sync v0.0.0-20200930132711-30421366ff76
+	golang.org/x/tools v0.0.0-20201002055958-0d28ed0cbe40
 	gopkg.in/yaml.v2 v2.3.0
-	k8s.io/client-go v0.18.6
+	k8s.io/client-go v0.18.9
```

Comment:
```suggestion
	k8s.io/client-go v0.19.2
```

Reply:
I wanted to wait until our cluster is updated.